pkg/controller/kabaneroplatform/kabaneroplatform_controller.go

```diff
@@ -33,7 +34,7 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileKabanero{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	return &ReconcileKabanero{client: mgr.GetClient(), scheme: mgr.GetScheme(), requeueDelayMapV1: make(map[string]RequeueData), requeueDelayMapV2: make(map[string]RequeueData)}
```

Comment:
I am not thrilled by how much code we're having to duplicate for the V1 and V2 collections.  Since the V1 collection was such a small blip in time, I'll open an issue to resolve this (either remove support for V1, or come up with another abstraction that doesn't require us to duplicate code).